crates/consensus/src/lib.rs

```diff
@@ -718,9 +718,9 @@ impl<
         }
     }
 
-    /// Get the current maximum height being tracked by the consensus engine.
-    pub fn current_height(&self) -> Option<u64> {
-        self.internal.keys().max().copied()
+    /// Get the heights currently being tracked by the consensus engine.
+    pub fn incomplete_heights(&self) -> HashSet<u64> {
+        self.internal.keys().copied().collect()
```

Comment:
This should also be reverted. Looking at it's use below, there's no need for this breaking change.

Reply:
Hmm, yes, the conversion was done by the caller - but it's still an odd interface, failing to provide current height just because all heights are momentarily finalized... Are you planning to use it?